src/shared/streak-celebration/StreakCelebrationModal.jsx

```diff
@@ -42,9 +42,12 @@ function getRandomFactoid(intl, streakLength) {
   return factoids[Math.floor(Math.random() * (factoids.length))];
 }
 
-async function calculateVoucherDiscount(voucher, sku, username) {
-  const urlBase = `${getConfig().ECOMMERCE_BASE_URL}/api/v2/baskets/calculate`;
-  const url = `${urlBase}/?code=${voucher}&sku=${sku}&username=${username}`;
+async function getDiscountCodeInfo(code, courseId) {
```

Comment:
@NawfalAhmed Add a comment on this function that explains why we have introduced this change for edx.org only.